packages/core/content-releases/server/src/controllers/release.ts

```diff
@@ -0,0 +1,51 @@
+import type Koa from 'koa';
+import { errors } from '@strapi/utils';
+import { RELEASE_MODEL_UID } from '../constants';
+import { validateCreateRelease } from './validation/release';
+
+const { ApplicationError } = errors;
+
+const releaseController = {
+  async find(ctx: Koa.Context) {
+    const { user } = ctx.state;
+
+    // Releases can only be find by super admins until we figure out how to handle permissions
+    if (!strapi.admin.services.role.hasSuperAdminRole(user)) {
+      throw new ApplicationError('Content Releases is a superadmin only feature');
+    }
+
+    const results = await strapi.entityService.findMany(RELEASE_MODEL_UID, {
+      populate: {
+        actions: {
+          count: true,
+        },
+      },
+    });
+
+    ctx.body = {
+      data: results,
+    };
```

Comment:
I don't see any way we could expose sensitive data, but just in case it's added in the future, we normally sanizite the output of our controllers. I can provide examples if you need them , I am on the phone rn

Reply:
You're right @Marc-Roig, I push this and also added pagination